celery_prometheus_exporter.py

```diff
@@ -360,13 +360,16 @@ def main():  # pragma: no cover
     w.start()
 
     if opts.queue_list:
+
+        # CLI argument will likely be a string nested in a list
```

Comment:
Wouldn't the opts.queue_list be a string?  As it is coming directly from the CLI?